backendless.d.ts

```diff
@@ -320,6 +320,8 @@ declare module Backendless {
 
         function resendEmailConfirmation(email: string): Promise<void>;
 
+        function createEmailConfirmation(email: string): Promise<void>;
```

Comment:
I believe this method should return something

Reply:
resolved